portfolio/pom.xml

```diff
@@ -34,9 +34,9 @@
       <version>1.9.59</version>
     </dependency>
     <dependency>
-      <groupId>com.google.appengine</groupId>
-      <artifactId>appengine-api-1.0-sdk</artifactId>
-      <version>1.9.59</version>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-language</artifactId>
+      <version>1.55.0</version>
```

Comment:
Hey I am doing the same exercise -- why is com.google.appengine removed from the dependency list?

Reply:
I realized I had the same dependency twice in one file, so i just replaced it.

Reply:
ah I see I see